atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -102,7 +102,10 @@ def next_release_general(self, component, version, release, release_label,
     def get_next_release_standard(self, component, version):
         build_info = {'name': component, 'version': version}
         self.log.debug('getting next release from build info: %s', build_info)
-        next_release = self.get_patched_release(self.xmlrpc.getNextRelease(build_info))
+        try:
+            next_release = self.get_patched_release(self.xmlrpc.getNextRelease(build_info))
+        except koji.BuildError:
```

Comment:
Could we add a comment on why we would reach a koji buildError here?

Reply:
added